src/themes/mixins.js

```diff
@@ -0,0 +1,174 @@
+// Let's start by generating a type scale, along with line-heights and small-caps modifiers.
+// These functions will be used by our type styles below.
+
+// In order to get a whole-integer pixel number, we multiply by ten and round,
+// then divide by ten again to get a rem value that works with our base size setting.
+const calculateFontSize = ({ starterSize, scaleModifier, sizeNumber }) => {
+  const newSize =
+    Math.round(starterSize * 10 * scaleModifier ** sizeNumber) / 10;
```

Comment:
Just checking, but this is [meant to be an exponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Exponentiation_(**)), right? It makes sense, but just making sure 😄

Reply:
IT SURE IS! I had to learn all kinds of mathy stuff in JS to make this do what I wanted.